jenkins/Jenkinsfile.linux

```diff
@@ -63,7 +63,7 @@ def getAgentLabel(String arch) {
     return 'linux && arm64 && 4x'
```

Comment:
Do you want to remove 4x here as well?

Reply:
Since Leticia just updated our x86 agents with more space, I've reverted this back to using the 4x agents instead!